content/home/usecases/usecases/usecase2.md

```diff
@@ -2,6 +2,6 @@
 title: "usecases"
 name : media
 heading : Media Storage
-isActive : true
+isActive : false
```

Comment:
what does this do?

Reply:
toggles active links in the "Tardigrade Cloud Object Storage Use Cases" section